edge-modules/ModuleLib/AnalyzerClient.cs

```diff
@@ -0,0 +1,504 @@
+﻿//----------------------
+// <auto-generated>
+//     Generated using the NSwag toolchain v13.1.3.0 (NJsonSchema v10.0.27.0 (Newtonsoft.Json v11.0.0.0)) (http://NSwag.org)
```

Comment:
I haven't used NSwag tool; what is the input for this tool?

Reply:
```nswag swagger2csclient /InjectHttpClient:false```